docs/scheduler/filter.md

```diff
@@ -101,7 +101,7 @@ new containers.
 Continuing the previous example, assuming your cluster with `node-1` and
 `node-2`, you can run a MySQL server container on the cluster.  When you run the
 container, you can use a `constraint` to ensure the database gets good I/O
-performance. You do this by by filter for nodes with flash drives:
+performance. You do this by filter for nodes with flash drives:
```

Comment:
according to @moxiegirl :pray: please use `by filtering for nodes` as @abronan suggested